content/administrators/people/roll-groups

```diff
@@ -0,0 +1,3 @@
+The Roll Groups module allows you to browse through the roll groups (also known as Form Groups in UK English) in your school. In the main table, you can see the names of any form tutors, what room they are based in, and how many students are in each roll group. There is also a View button that takes you to an overview of each individual roll group.
```

Comment:
Hi Harry, it looks like this file was meant to be replaced by the second one, but it was missing the md file extension, so a new file was created. Can you remove this one and push your changes to update your PR. Thanks!

Reply:
Hi, Sandra! I just made a new commit on the same branch that removes this file. Thanks for flagging it up!